src/app/pages/CreateConnectorPage/StepCoreConfiguration.tsx

```diff
@@ -66,42 +116,81 @@ export const StepCoreConfiguration: FC = () => {
-                  <TextInput
-                    value={serviceAccount.clientSecret}
-                    type={'password'}
-                    onChange={(clientSecret) =>
-                      onSetServiceAccount({ ...serviceAccount, clientSecret })
+            <FormGroup
+              label={t('clientId')}
+              isRequired
+              fieldId="clientId"
+              className="pf-u-mb-0"
+            >
+              <TextInput
+                value={serviceAccount.clientId}
+                onChange={(clientId) =>
+                  onSetServiceAccount({ ...serviceAccount, clientId })
+                }
+                id="clientId"
+              />
+            </FormGroup>
+            <FormGroup
+              label={t('clientSecret')}
+              isRequired
+              fieldId="clientSecret"
+              className="pf-u-mb-0"
+              helperText={
+                duplicateMode ? t('credentialDuplicateFieldHelpText') : ''
+              }
+            >
+              <TextInput
+                value={serviceAccount.clientSecret}
+                type={'password'}
+                onChange={(clientSecret) =>
+                  onSetServiceAccount({ ...serviceAccount, clientSecret })
+                }
+                id="clientSecret"
+              />
+            </FormGroup>
+            <FormGroup fieldId="saConfiguredConfirmation">
+              <Checkbox
+                label={
+                  <Trans
+                    i18nKey={'serviceAccountInstructionsConfirmationLabel'}
+                  >
+                    I have set the Kafka instance to allow access for this
+                    service account.
+                  </Trans>
+                }
+                description={
+                  <Trans
+                    i18nKey={
+                      'serviceAccountInstructionsConfirmationDescription'
                     }
-                    id="clientSecret"
-                  />
-                </FormGroup>
-              </>
-            )}
+                  >
+                    I configured the service account as described in{' '}
+                    <Button
+                      variant={ButtonVariant.link}
+                      isSmall
+                      isInline
+                      component={'a'}
+                      href={t('createServiceAccountForKafkaGuideLink')}
+                      target={'_blank'}
+                      ouiaId={'description-service-account-guide-link'}
+                    >
+                      Setting permissions for a service account in a Kafka
+                      instance in OpenShift Streams for Apache Kafka.
+                    </Button>
+                  </Trans>
+                }
+                id="saConfiguredConfirmation"
+                name="saConfiguredConfirmation"
+                ouiaId={'sa-configured-confirmation-checkbox'}
+                data-testid={'sa-configured-confirmation-checkbox'}
```

Comment:
@mmuzikar I added this as the other option for tests, just needs to be checked to continue through the wizard.